tests/integration/models/release.spec.js

```diff
@@ -276,6 +276,44 @@ describe('Release Model', function () {
 					});
 				});
 			});
+
+			describe('balena.model.release.setIsInvalidated()', function () {
+				parallel('', function () {
+					releaseRetrievalFields.forEach((field) => {
+						it(`should validate a release by ${field}`, async function () {
+							const draftRelease = testReleaseByField[field];
+							await balena.models.release.setIsInvalidated(
+								draftRelease[field],
+								false,
+							);
+							const validatedRelease = await balena.models.release.get(
+								draftRelease.id,
+							);
+							expect(validatedRelease).to.deep.match({
+								id: draftRelease.id,
+								commit: draftRelease.commit,
+								is_invalidated: false,
+							});
+						});
+
+						it(`should invalidate a release by ${field}`, async function () {
+							const draftRelease = testReleaseByField[field];
+							await balena.models.release.setIsInvalidated(
+								draftRelease[field],
+								true,
+							);
+							const invalidatedRelease = await balena.models.release.get(
+								draftRelease.id,
+							);
+							expect(invalidatedRelease).to.deep.match({
+								id: draftRelease.id,
+								commit: draftRelease.commit,
+								is_invalidated: false,
+							});
+						});
+					});
+				});
```

Comment:
There is obviously a race condition when running these tests in parallel, right :) ?
(plus you are checking for `is_invalidated: false` in both tests)
Try the following:
```suggestion
				releaseRetrievalFields.forEach((field) => {
					parallel('', function () {
							it(`should invalidate a release by ${field}`, async function () {
								const release = testReleaseByField[field];
								await balena.models.release.setIsInvalidated(
									release[field],
									true,
								);
								const invalidatedRelease = await balena.models.release.get(
									release.id,
									{ $select: 'is_invalidated' }
								);
								expect(invalidatedRelease).to.deep.match({
									is_invalidated: true,
								});
							});
						});
					});

					parallel('', function () {
						it(`should validate a release by ${field}`, async function () {
							const release = testReleaseByField[field];
							await balena.models.release.setIsInvalidated(
								release[field],
								false,
							);
							const validatedRelease = await balena.models.release.get(
								release.id,
								{ $select: 'is_invalidated' }
							);
							expect(validatedRelease).to.deep.match({
								is_invalidated: false,
							});
						});
					});
				});
```

Reply:
Didn't notice the `is_invalidated: false` 😖, thanks for catching that
Tried your suggestion locally and the tests passed :)